### PR TITLE
Add link checker

### LINKED ISSUE #89: Automate broken link checking
## What do you need?

An easier way to check for broken links.

Aware this may require different tools for internal and external links.

### PR DESCRIPTION
**Pull request by bjgordon**
## Description

Fixes #89 

This PR adds checking of internal links to the CI pipeline in `bin/citest.sh`, and fixes all the errors it found. The biggest change was how to handle missing local nav links, so to keep things simple I've moved some of the single page collections just to regular pages in `_pages`.
CI does not test external links, as it would be annoying to have temporary third party server outages stop the build. There is a bash script at `bin/checkexternal.sh` which will test external links which can be manually ran locally when desired. The only external error failure I currently get is to http://guides.service.gov.au/content-guide

## Definition of Done

- [ ] Acceptance testing (for code changes)
- [ ] Draft reviewed by other Content Designer
- [x] Changelog updated N/A
- [ ] PR approved by Libby
- [ ] Related GitHub issues closed
- [ ] Cloud apps deleted

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -115,61 +115,38 @@ highlighter: rouge
...
-  how-to-use-guide:
-    title: "How to use this guide"
-    description: "About this Content Guide and how you can use it."
-    order: 0
-    output: true
   content-structure:
     title: "Content structure"
     description: "The basic building blocks for crafting good content."
     order: 1
-    output: true
```
Comment (bjgordon) [inline _config.yml]: I removed these outputs, as the resulting pages weren't used as far as I could see and the link checker was failing them (i.e. the content in some of the collection items was expecting to be on the collection index page)

File: _includes/header.liquid [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <nav class="skip-to">
   <a href="#content">Skip to main content</a>
-  <a href="#nav">Skip to local navigation</a>
+  {% comment %} TODO is there a better way of deciding whether to show the local nav skip link? {% endcomment %}
```
Comment (bjgordon) [inline _includes/header.liquid]: Sorry this is a bit of a hack - seemed the easiest way to determine whether the page would have a local nav.

File: _config.yml (hunk 1 above)
Comment (joolswood) [inline _config.yml]: Interesting this seems to have maybe fixed another little bug I was going to hopefully fix: the `Privacy statement` link was showing up in the `Content` column of the footer as well as under Feedback (the correct place). Yay!

**Comment by joolswood** [comment]
Hey this is great @bjgordon thanks heaps! Can you verify this is all good @dominikwilkowski or @simonschwartz ?

**Comment by joolswood** [comment]
Hey @dominikwilkowski have given Brendan ok to merge this.

**Comment by dominikwilkowski** [comment]
I'm sure its awesome :)

**Pull request merged**